format module values in gce_net module documentation

dminca opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
The values passed to the module weren't being highlighted properly in
the documentation page.

As a result, use markup code format for the values provided in order to
be spotted easily on the page.

<!--- If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module, plugin, module or task -->
`gce_net`

##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```

```


##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful. -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

dminca committed:
format code in gce_net module

The values passed to the module weren't being highlighted properly in
the documentation page.

As a result, use markup code format for the values provided in order to
be spotted easily on the page.

Signed-off-by: Daniel Andrei Minca <mandrei17@gmail.com>

dminca committed:
separate phrases

Signed-off-by: Daniel Andrei Minca <mandrei17@gmail.com>

gundalow reviewed (commented):
For module options it's `I(foo)`
http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#formatting-functions

dminca committed:
apply review fix for module options

According to http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#formatting-functions

Signed-off-by: Daniel Andrei Minca <mandrei17@gmail.com>
